docs/how-to-build.md

```diff
@@ -75,7 +75,9 @@ but is not official and does not provide the same assurance as a command-line bu
 The additional requirements are:
 
 1. Visual Studio 2022 (17.0 or later)
-2. Install the ".NET Framework 4.6.2 targeting pack" individual component (via the Visual Studio installer)
+2. Install the following individual component via the Visual Studio installer:
+   * .NET Framework 4.7.2 targeting pack

```

Comment:
I couldn't find this thing! It seemed to me it came with VS. But you are better at this than I am.

Reply:
Is it now the Developer Pack? 

https://dotnet.microsoft.com/en-us/download/dotnet-framework/thank-you/net472-developer-pack-offline-installer

Reply:
Ah, no. The developer pack bundles the targeting pack!

Reply:
In the "Individual components" tab:
![image](https://github.com/user-attachments/assets/330b4329-f373-4720-a524-f1ccb6565b7e)

Reply:
Thanks! I never would've found!